<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1077</width>
    <height>689</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Qlossary</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Search: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_search">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>271</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Project:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_mainfrm_projectslst">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>241</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtdt_term_desc">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>311</width>
      <height>541</height>
     </rect>
    </property>
    <property name="documentTitle">
     <string/>
    </property>
    <property name="undoRedoEnabled">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QListWidget" name="lst_terms">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>241</width>
      <height>541</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1077</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="mnFile_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="mn_Help_About_Qlossary"/>
   </widget>
   <widget class="QMenu" name="menu_Project">
    <property name="title">
     <string>&amp;Projects</string>
    </property>
    <widget class="QMenu" name="menuNew_Language">
     <property name="title">
      <string>&amp;Languages</string>
     </property>
     <addaction name="mnProject_ViewLanguages"/>
     <addaction name="mnProject_NewLanguage"/>
    </widget>
    <widget class="QMenu" name="menuProjects">
     <property name="title">
      <string>Projects</string>
     </property>
     <addaction name="mnProject_New"/>
     <addaction name="mnProject_ViewProjects"/>
    </widget>
    <addaction name="menuNew_Language"/>
    <addaction name="menuProjects"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="mnView_Languages"/>
    <addaction name="mnView_Projects"/>
    <addaction name="mnView_NewTerm"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_Project"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dck_projects">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>260</width>
     <height>315</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>260</width>
     <height>315</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Projects</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_6">
    <widget class="QTabWidget" name="tab_projects">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>241</width>
       <height>281</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab_proj_Actives">
      <attribute name="title">
       <string>Actives</string>
      </attribute>
      <widget class="QPushButton" name="btn_projRemove">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>91</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_projSelect">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>220</y>
         <width>92</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
      <widget class="QListWidget" name="lst_projActives">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>221</width>
         <height>201</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_projTrash">
      <attribute name="title">
       <string>Trash</string>
      </attribute>
      <widget class="QListWidget" name="lst_projTrash">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>221</width>
         <height>191</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_projRemove_trash">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>92</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_projActivate_trash">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>210</y>
         <width>92</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Activate</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_projNew">
      <attribute name="title">
       <string>New</string>
      </attribute>
      <widget class="QPushButton" name="btn_projAdd">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>210</y>
         <width>92</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Project name:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ln_projName">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>221</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Original Language:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cmb_projOrigLang">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>211</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="cmb_projTargLang">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>211</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Target Language: </string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dck_languages">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>260</width>
     <height>315</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>260</width>
     <height>315</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="allowedAreas">
    <set>Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Languages</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_7">
    <widget class="QTabWidget" name="tab_languages">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>231</width>
       <height>271</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="usesScrollButtons">
      <bool>true</bool>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab_langsActives">
      <attribute name="title">
       <string>Actives</string>
      </attribute>
      <widget class="QListWidget" name="lst_langActives">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>211</width>
         <height>191</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_langRemove_actvs">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>92</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_langsTrash">
      <attribute name="title">
       <string>Trash</string>
      </attribute>
      <widget class="QListWidget" name="lst_langTrash">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>201</width>
         <height>192</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_langRemove_trash">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>92</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_langActivate_trash">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>210</y>
         <width>92</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Activate</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_langsNew">
      <attribute name="title">
       <string>New</string>
      </attribute>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Language: </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>61</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>Isocode: </string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ln_langName">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>201</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="ln_langIsocode">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>201</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_langAdd">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>180</y>
         <width>92</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dck_newterm">
   <property name="minimumSize">
    <size>
     <width>220</width>
     <height>630</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>220</width>
     <height>630</height>
    </size>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>New Term</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Original Term: </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ln_new_origterm">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>201</width>
       <height>25</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Original term</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Target Term:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ln_new_targterm">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>201</width>
       <height>25</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Target term</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>112</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Term type: </string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmb_new_termType">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>201</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>112</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Project: </string>
     </property>
    </widget>
    <widget class="QComboBox" name="cmb_newtermprojects">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>201</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Context: </string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txt_new_termContext">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>201</width>
       <height>261</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_clearTerm">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>560</y>
       <width>92</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_addTerm">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>560</y>
       <width>92</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="mnFile_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="mn_Help_About_Qlossary">
   <property name="text">
    <string>&amp;About Qlossary</string>
   </property>
  </action>
  <action name="mnProject_New">
   <property name="text">
    <string>&amp;New Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="mnProject_ViewLanguages">
   <property name="text">
    <string>&amp;View Languages</string>
   </property>
  </action>
  <action name="mnProject_NewLanguage">
   <property name="text">
    <string>&amp;New Language</string>
   </property>
  </action>
  <action name="mnProject_ViewProjects">
   <property name="text">
    <string>&amp;View Projects</string>
   </property>
  </action>
  <action name="mnView_Languages">
   <property name="text">
    <string>&amp;Languages</string>
   </property>
  </action>
  <action name="mnView_Projects">
   <property name="text">
    <string>&amp;Projects</string>
   </property>
  </action>
  <action name="mnView_NewTerm">
   <property name="text">
    <string>&amp;New Term</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tab_languages</tabstop>
  <tabstop>ln_langName</tabstop>
  <tabstop>ln_langIsocode</tabstop>
  <tabstop>btn_langAdd</tabstop>
  <tabstop>lst_langActives</tabstop>
  <tabstop>btn_langRemove_actvs</tabstop>
  <tabstop>lst_langTrash</tabstop>
  <tabstop>btn_langActivate_trash</tabstop>
  <tabstop>btn_langRemove_trash</tabstop>
  <tabstop>tab_projects</tabstop>
  <tabstop>ln_projName</tabstop>
  <tabstop>cmb_projOrigLang</tabstop>
  <tabstop>cmb_projTargLang</tabstop>
  <tabstop>btn_projAdd</tabstop>
  <tabstop>lst_projActives</tabstop>
  <tabstop>btn_projSelect</tabstop>
  <tabstop>btn_projRemove</tabstop>
  <tabstop>lst_projTrash</tabstop>
  <tabstop>btn_projActivate_trash</tabstop>
  <tabstop>btn_projRemove_trash</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>mnFile_Quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
